velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -4014,3 +4048,43 @@ TEST_F(DateTimeFunctionsTest, toISO8601Date) {
   EXPECT_EQ("-3492-10-05", toISO8601("-3492-10-05"));
   EXPECT_EQ("-0653-07-12", toISO8601("-653-07-12"));
 }
+
+TEST_F(
+    DateTimeFunctionsTest,
+    timestampAtTimezoneTestTimestampWithTimezoneInput) {
+  EXPECT_EQ(
+      TimestampWithTimezone(1500101514, util::getTimeZoneID("America/Boise")),
+      timestampWithTimezoneAtTimezone(
```

Comment:
call the function `atTimezone()` to follow the pattern used for other functions?

Reply:
Renamed.